### DISCUSSION TITLE
Is it possible to use this library to create a custom voice for Windows 11 speech synthesizer?

### DISCUSSION THREAD

**Discussion by siddjain** [coqui-ai/TTS#3838] [General Q&A]
Is it possible to use this library to create a custom voice for Windows 11 speech synthesizer (see below for what I mean)? How?
<img width="380" alt="image" src="https://github.com/user-attachments/assets/7d5adaee-c531-4b5d-9a27-170234cbc5d3">

I want to add my voice to the Settings above.

**Comment by FrontierDK** [comment]
Windows uses SAPI, not neural networks. That being said, I don't know if someone has made something similar to what Amazon has made (a component in the "middle" which orders the audio files). More [here](https://aws.amazon.com/about-aws/whats-new/2019/09/amazon-polly-voices-available-in-windows-applications/)
